File: hw/collahuazo/u2/hw14MongoDBAndGUI/InventoryhubJavascript/src/models/product.js
export class Product {
  constructor(id, name, unitPrice, stock) {
    this.id = Number(id);
    this.name = String(name);
    this.unitPrice = Number(unitPrice);
    this.stock = Number(stock);
  }

  toDocument() {
    return {
      id: this.id,
      name: this.name,
      unitPrice: this.unitPrice,
      stock: this.stock,
    };
  }
}

File: hw/collahuazo/u2/hw14MongoDBAndGUI/InventoryhubPython/models/product.py
from dataclasses import dataclass


@dataclass
class Product:
    id: str
    name: str
    unit_price: float
    stock: int

    def to_dict(self) -> dict:
        return {
            "id": self.id,
            "name": self.name,
            "unitPrice": self.unit_price,
            "stock": self.stock,
        }
